tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -70,12 +70,16 @@ export const clickResource = async ({
   for (const name of paths) {
     const resource = await page.locator(util.format(resourceNameSelector, name))
     const itemId = await resource.locator(fileRow).getAttribute('data-item-id')
-    await Promise.all([
-      resource.click(),
-      page.waitForResponse(
-        (resp) => resp.url().endsWith(encodeURIComponent(name)) || resp.url().endsWith(itemId)
-      )
-    ])
+    try {
+      await Promise.all([
+        resource.click(),
+        page.waitForResponse(
+          (resp) => resp.url().endsWith(encodeURIComponent(name)) || resp.url().endsWith(itemId)
+        )
+      ])
+    } catch (e) {
+      throw new Error(`Could not open resource ${name}`)
```

Comment:
```suggestion
      throw new Error(`Could not open resource ${name}\n. Error: ${e}`)
```
showing what the real error was will help while debugging